apps/picsa-tools/monitoring-tool/src/app/components/access-code-dialog/access-code-dialog.component.html

```diff
@@ -0,0 +1,35 @@
+<h2 mat-dialog-title>{{ 'Restricted Form' | translate }}</h2>
+<div mat-dialog-content>
+  <p>
+    {{ 'This form requires an access code to unlock' | translate }}: <strong>{{ data.formTitle }}</strong>
+  </p>
+
+  <mat-form-field style="width: 100%" [class.error]="showError()">
+    <mat-label>{{ 'Access Code' | translate }}</mat-label>
+    <input
+      matInput
+      [(ngModel)]="accessCode"

```

Comment:
nit(non-blocking): angular has also now added a signal alternative for ngModel, which would be good to use instead:
https://angular.dev/guide/components/inputs#model-inputs